src/angular-youtube-embed.js

```diff
@@ -1,3 +1,7 @@
+if (!angular && require){
+    var angular = require('angular');
+}
```

Comment:
along those same lines, it seems like it's better to just let people get their own `angular`. having the code throw that `angular` is `undefined` seems appropriate

also, how does `browserify` and the like handle a `require` inside a conditional?